programs/ziti-edge-tunnel/package/CPackPackage.cmake

```diff
@@ -59,7 +59,7 @@ set(SYSTEMD_UNIT_FILE_NAME "${SYSTEMD_SERVICE_NAME}.service")
 set(SYSTEMD_EXECSTARTPRE "${SYSTEMD_SERVICE_NAME}.sh")
 set(SYSTEMD_ENV_FILE "${SYSTEMD_SERVICE_NAME}.env")
 
-set(ZITI_IDENTITY_DIR "${CPACK_ETC_DIR}/identity")
+set(ZITI_IDENTITY_DIR "${CPACK_ETC_DIR}/identities")
```

Comment:
I like "identities" better, but we need to provide an upgrade path for at least one minor release, err one micro release since we're still in magic zero version territory. That is, for the version in which this change ships we need a symlink or behavior to allow identities in the old dir to continue to function when the package upgrade is installed. Else, it'll bomb!

Reply:
I think "need" is a bit strong given the age of packaging,and fact it is not yet advertised. I thought about adding something to address that, but ultimately concluded it would unnecessarily polute the code.

Reply:
Agreed. As long as we're treating the repo distribution channels as a contract I'm good. These are bleeding edge builds so I reckon they can break compat if expedient.